<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RepairDriver</class>
 <widget class="QWidget" name="RepairDriver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RepairDriver</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget
{
	background-color: rgb(255, 255, 255);
    border-raduis: 4px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="titleBar" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>44</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>44</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#titleBar
{
    background-color: rgb(238, 238, 238);
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">color: rgb(48, 48, 48);
font: 75 12pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>LJtool驱动组件修复</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closeBtn">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>关闭</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#closeBtn {
	border-image: url(:/res/dialog_close.png) 0 0 96 0;
}

QPushButton#closeBtn:hover {
	border-image: url(:/res/dialog_close.png) 32 0 64 0;
}

QPushButton#closeBtn:pressed {
	border-image: url(:/res/dialog_close.png) 64 0 32 0;
}

QPushButton#closeBtn:disabled {
	border-image: url(:/res/dialog_close.png) 96 0 0 0;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>90</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="m_iconLab">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/res/repairIcon.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>8</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">font-size: 18px;
font-weight: 800;
font-family:Microsoft YaHei;</string>
             </property>
             <property name="text">
              <string>检查电脑环境</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="minimumSize">
                <size>
                 <width>29</width>
                 <height>19</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>29</width>
                 <height>19</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border-image: url(:/res/tipiconpng.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>设备第一次连接需要安装驱动程序，请耐心等待安装完成</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="m_pUpdateBtn">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>刷新</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#m_pUpdateBtn{
border-image:url(:/res/update_icon.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	border: none;
	background-color: rgb(9, 200, 123);
}
QPushButton:hover
{
	background-color: rgb(7, 168, 114);
}
QPushButton:pressed
{
	background-color: rgb(7, 171, 105);
}</string>
           </property>
           <property name="text">
            <string>一键修复</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_head" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>28</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(231, 245, 255);</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="styleSheet">
               <string notr="true">color: #444444;</string>
              </property>
              <property name="text">
               <string>检测内容描述</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #444444;</string>
              </property>
              <property name="text">
               <string>状态</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_appleUSBConnect" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>38</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>已连接的USB设备</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_noconnect">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string>未检测到设备连接</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_iPhone">
              <property name="text">
               <string>iPhone</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_appleMDSupoort64" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>38</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Apple Mobile Device Support64bit</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_amds64_install">
              <property name="styleSheet">
               <string notr="true">color: rgb(68, 161, 234);</string>
              </property>
              <property name="text">
               <string>已安装</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_amds64_no">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string>未安装</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_appleMDSupoort32" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>38</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Apple Mobile Device Support</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_amds_install">
              <property name="styleSheet">
               <string notr="true">color: rgb(68, 161, 234);</string>
              </property>
              <property name="text">
               <string>已安装</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_amds_no">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string>未安装</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_appleAppSupport32" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>38</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Apple 应用程序支持（32位）</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_appsupport32_install">
              <property name="styleSheet">
               <string notr="true">color: rgb(68, 161, 234);</string>
              </property>
              <property name="text">
               <string>已安装</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_appsupport32_no">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string>未安装</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_appleAppSupport64" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>38</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Apple 应用程序支持（64位）</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_appsupport64_install">
              <property name="styleSheet">
               <string notr="true">color: rgb(68, 161, 234);</string>
              </property>
              <property name="text">
               <string>已安装</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_appsupport64_no">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string>未安装</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_appleMDService" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>38</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Apple Mobile Device服务</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_have_start">
              <property name="styleSheet">
               <string notr="true">color: rgb(68, 161, 234);</string>
              </property>
              <property name="text">
               <string>已启动</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_no_start">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string>未启动</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>17</width>
             <height>76</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
